app/views/reports/performance_profiles.html.erb

```diff
@@ -0,0 +1,131 @@
+    <h1 class="govuk-heading-l">
+      Trainees who studied in the <%= @previous_academic_cycle_label %> academic year
+    </h1>
+    <p class="govuk-body">
+      You can use this report to help you sign off your data for the initial
+      teacher training (ITT) performance profiles publication.
+    </p>
+    <p class="govuk-body">
+      You must check it, fix any mistakes and sign it off by 4pm on
+      <%= @sign_off_date %>.
+    </p>
+    <p class="govuk-body">
+      This export includes all trainees who:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        have been registered with the Department for Education
+      </li>
+      <li>
+        studied in the <%= @previous_academic_cycle_label %> academic year
+      </li>
+    </ul>
+
+    <h2 class="govuk-heading-m">
+      Check your trainee data
+    </h2>
+    <p class="govuk-body">
+      You need to check that:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        all your <%= @previous_academic_cycle_label %> trainees are listed, including any who deferred or
+        withdrew from the course
+      </li>
+      <li>
+        the number of trainees awarded qualified teacher status (QTS) or early
+        years teacher status (EYTS) is accurate
+      </li>
+    </ul>
+    <p class="govuk-body">
+      For each trainee you need to check:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        course details
+      </li>
+      <li>
+        status
+      </li>
+      <li>
+        the date they were awarded QTS, deferred or withdrew from the course,
+        unless they are still studying
+      </li>
+    </ul>
+
+    <h2 class="govuk-heading-m">
+      Fix mistakes
+    </h2>
+    <p class="govuk-body">
+      If the trainee has been awarded or withdrawn, email
+      <a class="govuk-link" href="mailto:becomingateacher@digital.education.gov.uk?subject=Accessibility%20issues%20">becomingateacher@digital.education.gov.uk</a>
+      .
+    </p>
+    <p class="govuk-body">
+      If a trainee is still studying or has deferred, how you fix a mistake
+      depends on whether you registered the trainee using HESA or this service.
+    </p>
+
+    <h3 class="govuk-heading-s">
+      Trainees registered using HESA
+    </h3>
+    <p class="govuk-body">
+      TBD
+    </p>
+
+    <h3 class="govuk-heading-s">
+      Trainees registered using this service
+    </h3>
+    <p class="govuk-body">
+      If the trainee is still studying or has deferred, you should update their
+      details with the correct information.
+    </p>
+
+    <h2 class="govuk-heading-m">
+      Sign off your trainee data
+    </h2>
+    <p class="govuk-body">
+      Once you’ve checked your data, a senior person from your organisation
+      should sign it off
+      <a href=<%=@sign_off_url %> class="govuk-link" rel="noreferrer noopener" target="_blank">using this form (opens in a new tab)</a>
```

Comment:
Need URL for form sign off @claire-hughez @johndoates

Reply:
now: https://forms.gle/AtAvteRTKJu3mbXS7